toolchain/sem_ir/ids.h

```diff
@@ -480,16 +456,21 @@ struct IntKind : public IdBase, public Printable<IntKind> {
 constexpr IntKind IntKind::Unsigned = IntKind(0);
 constexpr IntKind IntKind::Signed = IntKind(1);
 
-// A float kind value
-struct FloatKind : public IdBase, public Printable<FloatKind> {
+// A float kind value.
+struct FloatKind : public IdBase<FloatKind> {
+  // Not used by `Print`, but for `IdKind`.
+  static constexpr llvm::StringLiteral Label = "float";
```

Comment:
To be consistent with my suggestion for `IntKind`:
```suggestion
  static constexpr llvm::StringLiteral Label = "float_kind";
```

Reply:
Done